server/routes/adminRoutes.js

```diff
@@ -293,4 +278,22 @@ router.get('/export/businesses/',
     }
 );
 
+if (process.env.NODE_ENV === 'development' || !process.env.NODE_ENV) {
```

Comment:
instead of repeating "development" everywhere, can we make this a global variable and use it across all the files?

Reply:
Great idea!

Reply:
great idea!